scripts/asr_language_modeling/ngram_lm/create_lexicon_from_arpa.py

```diff
@@ -1,76 +1,77 @@
-# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.

```

Comment:
Why is this file in the diff?

Reply:
Bit of a cheeky last minute add really. This file has already been committed to Nemo main, but I learned during this PR that we shouldn't ever use print statements in our scripts, so I fixed this script which is the only other one which had print statements (and is one I also wrote). So this just brings everything into line with Nemo's code standards. The reason it's all red/green is because it had Windows EOL and I changed them to Unix.